### ISSUE TITLE
README update

### ISSUE THREAD

**Issue by itisaby** [acm-iem/VirtualMouse#2]
Update the README file so that the new contributors get aware of what and where they need to contribute

**Comment by nkhandelwal07** [comment]
I can do this! Do you just want the Hacktober fest contribution information?

**Comment by yashpy** [comment]
hi itisaby,

Please assign this task to me, I can fix the issue,

Thanking you,

**Comment by itisaby** [comment]
@nkhandelwal07 yes but also it must be reusable so that even after hacktober fest people can understand what they need to contribute and where they can contribute.

**Comment by itisaby** [comment]
@yashpy I wish I could assign you but @nkhandelwal07 asked first so let him work on it, but if he is unable to fix it I will assign it to you for sure, till then you can take a look at other issues in other repos.
Keep hacking

**Comment by yashpy** [comment]
@itisaby yeah sure.

**Comment by nkhandelwal07** [comment]
@itisaby 
If @yashpy  wants to complete the issue then they can do it, since I also cannot work on it right away.

**Comment by yashpy** [comment]
sorry @itisaby, but it says readme pr are not included, its also shows excluded in my account,so I may not able to do another readme update,thanks please consider other contributors. Thanks @nkhandelwal07 for the consideration.

**Comment by nkhandelwal07** [comment]
Ok

**Comment by om-sharma03** [comment]
Hey @itisaby @itisdb May I work on this issue?

**Comment by itisaby** [comment]
@nkhandelwal07 Are you still working on this?

**Comment by itisaby** [comment]
@om-sharma03 yes you can work on this.

**Comment by nkhandelwal07** [comment]
It is fine someone else can work on it

**Comment by lonewolf235** [comment]
hey, @itisaby if someone is not working up on this issue I can do it. 
Can you assign me this task.

**Comment by lonewolf235** [comment]
If no one is working on it, can you assign this task to me?

**Comment by itisaby** [comment]
Alright @lonewolf235
